src/ImageListPage.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import ComponentPage from './ComponentPage.js';
+import HeaderBar from './HeaderBar.js';
+import {imagePath} from './constants';
+
+import './styles/ImageListPage.scss';
+
+function ImageListPage() {
+  return (
+    <div>
+      <HeaderBar />
+      <ComponentPage
+        hero={<ImageListHero />}
+        title='Image List'
+        description='Image lists display a collection of images in an organized grid.'
+        designLink='https://material.io/go/design-image-list'
```

Comment:
FYI this goes to a COMING SOON! page. Just wanna make sure that is intendend.

Reply:
ya thats right